libs/application/core/src/lib/messages.ts

```diff
@@ -442,6 +442,17 @@ export const coreErrorMessages = defineMessages({
       'Staðfesting á skuldleysi fékkst ekki úr gagnagrunni Fjársýslu ríkisins',
     description: '',
   },
+  missingJudicialAdministrationificateTitle: {
+    id: 'ndc.application:missingJudicialAdministrationificateTitle',
+    defaultMessage: 'Ekki tókst að staðfesta búsforræðisvottorð',
+    description: '',
+  },
+  missingJudicialAdministrationificateSummary: {
+    id: 'ndc.application:missingJudicialAdministrationificateSummary',
+    defaultMessage:
+      'Staðfesting á skuldleysi fékkst ekki úr gagnagrunni Dómstólasýslu ríkisins',
+    description: '',
+  },
```

Comment:
Any reason why these translation strings are in `application/core` instead of just the template?